### PR TITLE
Add CVE-2024-47850 to manual entries

### PR DESCRIPTION
**Pull request by dcaravel**
Largely mirrors changes made in: https://github.com/stackrox/scanner/pull/1657

### CODE REVIEW

**Comment by dcaravel** [comment]
Moving out of draft to trigger CI

File: ext/vulnsrc/manual/manual.go [hunk 1]
```diff
@@ -1514,6 +1514,388 @@ var Vulnerabilities = []database.Vulnerability{
...
+				"CVSSv3": map[string]interface{}{
+					"ExploitabilityScore": 3.9,
+					"ImpactScore":         3.6,
+					"Score":               7.5,
+					"Vectors":             "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
+				},
+			},
+		},
+	},
+	// Ubuntu 16.04
```
Comment (RTann) [inline ext/vulnsrc/manual/manual.go:1690]: Do you mind also updating the fixed in version for https://ubuntu.com/security/CVE-2024-47175 and https://ubuntu.com/security/CVE-2024-47176 ? sorry 😬

File: e2etests/testcase_test.go [hunk 2]
```diff
@@ -3696,7 +3696,7 @@ Bug Fix(es) and Enhancement(s):
 				NamespaceName: "rhel:8",
 				VersionFormat: "rpm",
 				Version:       "6.0.6-1.el8_6.x86_64",
-				FixedBy:       "6.0.32-1.el8_10",
+				FixedBy:       "6.0.35-1.el8_10",
```
Comment (RTann) [inline e2etests/testcase_test.go:3699]: not here, but in the rhel 8 cups-filters test, can you check for [RHSA-2024:7463](https://access.redhat.com/errata/RHSA-2024:7463), too?

**Review by RTann — APPROVED** [review]
Changed LGTM, so approving, but either here or in a followup can you account for the latest data which we did not have previously? Thanks

File: e2etests/testcase_test.go (hunk 2 above)
Comment (dcaravel) [reply e2etests/testcase_test.go:3699]: image

Test already there?

Comment (RTann) [reply e2etests/testcase_test.go:3699]: ah they added this CVE in after the fact. Cool

File: ext/vulnsrc/manual/manual.go (hunk 1 above)
Comment (dcaravel) [reply ext/vulnsrc/manual/manual.go:1690]: Also updated the scores for to match Ubuntus: https://ubuntu.com/security/CVE-2024-47176#impact-score

- Previous test had base score of 8.3
- Ubuntu page has base score of 7.5
- NVD has base score of 5.3

File: ext/vulnsrc/manual/manual.go [hunk 3]
```diff
@@ -994,10 +994,10 @@ var Vulnerabilities = []database.Vulnerability{
...
 				},
 				"CVSSv3": map[string]interface{}{
-					"ExploitabilityScore": 1.6,
-					"ImpactScore":         6.0,
-					"Score":               8.3,
-					"Vectors":             "CVSS:3.1/AV:N/AC:H/PR:N/UI:R/S:C/C:H/I:H/A:H",
+					"ExploitabilityScore": 3.9,
+					"ImpactScore":         3.6,
+					"Score":               7.5,
+					"Vectors":             "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
```
Comment (RTann) [inline ext/vulnsrc/manual/manual.go:1000]: I have no idea where they got this score from. I can't find it in their repo https://git.launchpad.net/ubuntu-cve-tracker/tree/active/ nor NVD https://nvd.nist.gov/vuln/detail/CVE-2024-47176

I wonder if this score changed? I can handle the other distros in a followup PR so this isn't blocked

**Review by RTann — APPROVED** [review]

**Pull request merged**